engine/drones/Gristle.scd

```diff
@@ -0,0 +1,20 @@
+// @infinitedigits
+// Gristle
+// a primal sawtooth
+
+{
+    arg hz=440, amp=0.02, amplag=0.02, hzlag=0.01;
```

Comment:
Were these left in on purpose? Seems like the upstream `amplag` and `hzlag` won't get passed into this synth.

Reply:
thanks, thats a mistake, I can remove those.